### PR TITLE
Added new keymap for KPrepublic BM60v2_poker

### PR DESCRIPTION
**Pull request by TheChilledDejw**
## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [ ] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* None

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/kprepublic/bm60hsrgb_poker/rev2/keymaps/TheChilledDejw/rules.mk [hunk 1]
```diff
@@ -0,0 +1,33 @@
+# MCU name
```
Comment (drashna) [inline keyboards/kprepublic/bm60hsrgb_poker/rev2/keymaps/TheChilledDejw/rules.mk:1]: Most of this should be removed.  Only the settings that need to be changed should be included here.

**Review by drashna — CHANGES_REQUESTED** [review]
Please rename the keymap folder to be all lower case.

**Comment by TheChilledDejw** [comment]
> Please rename the keymap folder to be all lower case.

Renamed, everything should be fine with the folders

**Comment by drashna** [comment]
It looks like you have changes to the submodules here.  These need to be reverted before the PR can be approved. 

Rebasing sometimes will fix this. Otherwise, you'd want to do something like this to fix them:
```sh
git checkout master
git submodule foreach 'git log -n1 --oneline'
# remember the first abcdef1234 on the submodules that have an issue
git checkout @@your-branch@@
# for each of the problematic repo's:
cd lib/@@name@@
git checkout abcdef1234
```

**Pull request closed without merging**